externs/olx.js

```diff
@@ -3135,6 +3135,16 @@ olx.interaction.DrawOptions.prototype.geometryName;
  */
 olx.interaction.DrawOptions.prototype.condition;
 
+/**
+ * A function that takes an {@link ol.MapBrowserEvent} and returns a boolean
+ * to indicate whether the current sketch being drawn should be updated. This
+ * can be used to either limit the scope of the feature being drawn, or to
+ * skip events caused by panning/zooming on a touch device. By default
+ * {@link ol.events.condition.always}, i.e. it is always updated.
+ * @type {ol.EventsConditionType|undefined}
+ * @api
+ */
+olx.interaction.DrawOptions.prototype.updateSketchCondition;
```

Comment:
The new option must also be added in the `olx.interaction.DrawOptions` `@typedef` above.

Reply:
Thanks for noticing, fixed by https://github.com/openlayers/openlayers/pull/7535/commits/b53d61d9ff8586813cf9f88945d39c17b1c24b13.